src/main/java/net/openhft/chronicle/queue/impl/single/Pretoucher.java

```diff
@@ -155,12 +140,10 @@ private void assignCurrentCycle() {
                 cycleChangedListener.accept(qCycle);
 
                 if (earlyAcquireNextCycle)
```

Comment:
Why we are only writing a message if this flag is set? I guess this line has to go and that isDebugEnabled the one that got to stay.